### PR TITLE
changelog: added changelog for 0.8.

### PR DESCRIPTION
**Pull request by moonming**
NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/iresty/apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
 - [0.6.0](#060)
 
+## 0.8.0
+> Released on 2019/09/30
+
+This release brings many new features, such as stream proxy, support MQTT protocol proxy,
+and support for ARM platform, and proxy rewrite plugin.
+
+### Core
+- :sunrise: **[support stream proxy](https://github.com/iresty/apisix/blob/master/doc/stream-proxy.md)**. [513](https://github.com/iresty/apisix/pull/513)
```
Comment (membphis) [inline CHANGELOG.md]: missing MQTT proxy

**Comment by membphis** [comment]
We can add a new chapter about `doc`.

**Comment by membphis** [comment]
We should also check the commit log of the dependent libraries, which will include some new features.

**Comment by membphis** [comment]
> We should also check the commit log of the dependent libraries, which will include some new features.

lua-resty-ipmatcher:

* support IPv6: IP white/black list, route.

resty-radixtree:

* allow to specify multiple `host`, `remote_addr` and `uri`.
* allow to define user-function to filter request.
* use `lua-resty-ipmatcher` instead of `lua-resty-iputils`, `lua-resty-ipmatcher` matches fast and support IPv6.

File: CHANGELOG_CN.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 这个版本带来很多新的特性，比如四层协议的代理, 支持 MQTT 协议代理，以及对 ARM 平台的支持, 和代理改写插件等。
 
 ### Core
+- :sunrise: **[增加单机模式](https://github.com/iresty/apisix/blob/master/doc/stand-alone-cn.md)**: 使用 yaml 配置文件来更新 APISIX 的配置，这对于kubernetes 更加友好。 [464](https://github.com/iresty/apisix/pull/464)
```
Comment (juzhiyuan) [inline CHANGELOG_CN.md]: `这对于kubernetes 更加友好` --> `这对于 kubernetes 更加友好`

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ This release brings many new features, such as stream proxy, support MQTT protoc
 and support for ARM platform, and proxy rewrite plugin.
 
 ### Core
+- :sunrise: **[support stand-alone mode](https://github.com/iresty/apisix/blob/master/doc/stand-alone-cn.md)**: using yaml to update configurations of APISIX, more friendly for kubernetes. [464](https://github.com/iresty/apisix/pull/464)
```
Comment (juzhiyuan) [inline CHANGELOG.md]: `more friendly to kubernetes` would be better

Comment (moonming) [reply CHANGELOG.md]: thx

**Pull request merged**